deepchecks/suites/README.md

```diff
@@ -14,38 +14,14 @@
 
 ## Using Existing CheckSuites
 
-### List of Prebuilt Suites
+### [List of Prebuilt Suites](./default_suites.py)
 
-[**Overall Suites**](./overall_suite.py)
-
-  - overall_suite - run all deepchecks checks, including checks for index and date
-  - overall_classification_suite - run all deepchecks checks for classification tasks with no index or date
-  - overall_regression_suite - run all deepchecks checks for regression tasks with no index or date
-  - overall_generic_suite - run all deepchecks checks that work regardless of task type with no index or date
-
-[**Distribution Suites**](./distribution_suite.py)
-
-  - data_distribution_suite - run all data distribution checks
-
-[**Integrity Suites**](./integrity_suite.py)
-
-  - single_dataset_integrity_suite - for a single dataset / dataframe
-  - comparative_integrity_suite - comparing two datasets / dataframes
-  - integrity_suite - includes both check types 
-
-[**Methodology Suites**](./methodology_suite.py)
-  - index_leakage_suite - for datasets with an index column
-  - date_leakage_suite - for datasets with a date column
-  - data_leakage_suite  - for all datasets
-  - leakage_suite - containing all three suites above
-  - overfit_suite - run all overfit checks
-  - methodological_flaws_suite - checks for all methodological flaws, including unused features
+  - single_dataset_integrity - Runs a set of checks that are meant to detect integrity issues within a single dataset.
+  - train_test_leakage - Runs a set of checks that are meant to detect data leakage from the training dataset to the test dataset.
+  - train_test_validation - Runs a set of checks that are meant to validate correctness of train-test split, including integrity, drift and leakage.
+  - model_evaluation - Runs a set of checks that are meant to test model performance and overfit.
+  - full_suite - Runs all deepchecks' checks.
```

Comment:
Not accurate - it runs many, but not necessarily all
(It's the "all you can get" scenario, but some checks may be irrelevant for this. e.g. when we'll have explainability checks, and various checks that requires more configurations like "segment performance")